js-challenge/bbs-api/main.js

```diff
@@ -190,6 +167,224 @@ const clickUsersEditSubmit = () => {
     }).catch(response => {
         console.log(response);
     });
-
 };
 buttonUsersEditSubmit.addEventListener('click', clickUsersEditSubmit);
+
+//threadpost
+const buttonthreadsPostSubmit = document.getElementById("threadsPostSubmit");
+const clickThreadSubmi = () => {
```

Comment:
単語のタイプミスですね、例えば誰かがソースコード全体をsubmitで検索したときにhitしないものがあるという事態は危険なので直しましょう。
```suggestion
const clickThreadSubmit = () => {
```